openhabian.conf.dist

```diff
@@ -62,6 +62,10 @@ debugmode=off
 # Valid arguments: "Adopt11", "Zulu8-32", "Zulu8-64", "Zulu11-32", or "Zulu11-64"
 java_opt=Zulu8-32
 
+# openHAB Log Viewer (frontail) theme
+# Valid arguments: "light", "dark"
+frontail_theme=light
```

Comment:
pls no underscore  (all lowercase)

add  these lines to build-image/*.conf, too